sixtyfps_compiler/passes/resolving.rs

```diff
@@ -64,6 +64,10 @@ fn resolve_expression(
                 Expression::from_binding_expression_node(node.clone(), &mut lookup_ctx)
             }
             SyntaxKind::TwoWayBinding => {
+                if lookup_ctx.property_type == Type::Invalid {
+                    assert!(diag.has_error());
```

Comment:
This assert could perhaps benefit from a comment, elaborating how untyped two-way bindings start with Type::Void, get processed by the alias type inference pass by changing to Type::Invalid and attempting resolution. Failure to resolution must produce an error.

Reply:
```suggestion
                    // An attempt to resolve this already failed when trying to resolve the property type
                    assert!(diag.has_error());
```